### PR TITLE
Don't offer a choice, if we really only have one option.

### PR DESCRIPTION
**Pull request by mikelambert**

### CODE REVIEW

**Comment by mikelambert** [comment]
Ping?

File: android/src/main/java/com/chirag/RNMail/RNMailModule.java [hunk 1]
```diff
@@ -86,13 +86,18 @@ public void mail(ReadableMap options, Callback callback) {
...
-    Intent chooser = Intent.createChooser(i, "Send Mail");
-    chooser.setFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
-
-    try {
-      reactContext.startActivity(chooser);
-    } catch (Exception ex) {
-      callback.invoke("error");
+    if (list.size() == 1) {
+      i.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
+      reactContext.startActivity(i);
```
Comment (chirag04) [inline android/src/main/java/com/chirag/RNMail/RNMailModule.java]: this can fail. you are missing a try catch.

**Comment by chirag04** [comment]
thanks 👍

**Comment by chirag04** [comment]
ok going to do a minor version bump instead of major. This should have been the default behavior

**Pull request merged**